File: vue/personel/src/models/PersonelModel.js
export default class Personel {
   

    tcKimlikNo;
    ad ;
    soyad ;
    maas ;
    dogumTarih ;
    adres;

 
    constructor(p_sicilNo,p_tckimlikNo,p_ad,p_soyad,p_maas,p_dogumTarih,p_adres) 
    {
      this.sicilNo=p_sicilNo;
      this.tcKimlikNo=p_tckimlikNo;
      this.ad=p_ad;
      this.soyad=p_soyad;
      this.maas=p_maas;
      this.dogumTarih=p_dogumTarih;
      this.adres=p_adres;
    }
         
}

File: NetCore/Api/Api/Models/PersonelModel.cs
using System.ComponentModel.DataAnnotations;

namespace Api.Models
{
    public class PersonelModel
    {
       
        public int SicilNo { get; set; }
        [Key]
        public Int64 TcKimlikNo { get; set; }
        public string Ad { get; set; }
        public string Soyad { get; set; }

        public double Maas { get; set; }
        public DateTime DogumTarih { get; set; }
        public string Adres { get; set; }
    }
}
